File: PaylocityChallenge.Frontend/src/dtos/partial/employee-summary-dto.ts
import { FamilyMemberDto } from "./family-member-dto";

export interface EmployeeSummaryDto extends FamilyMemberDto {
  annualBenefitsPremium: number;
  annualBenefitsDiscount: number;
  annualBenfitsSubtotal: number;
  annualTaxableIncome: number;
}

File: PaylocityChallenge.Api/Dtos/Partial/EmployeeSummaryDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PaylocityChallenge.Api.Dtos.Partial
{
    public class EmployeeSummaryDto : FamilyMemberDto
    {
        public decimal AnnualBenefitsPremium { get; set; }
        public decimal AnnualBenefitsDiscount { get; set; }
        public decimal AnnualBenefitsSubtotal { get; set; }
        public decimal AnnualTaxableIncome { get; set; }
    }
}
